test/dynamo/test_higher_order_ops.py

```diff
@@ -1127,16 +1115,17 @@ def false_fn(x):
 
                 return control_flow.cond(y, true_fn, false_fn, [x])
 
+        mod_for_eager = Foo()
+        mod_for_eager(torch.tensor(True), torch.tensor(5))
+
```

Comment:
Btw, we shouldn't have any users of cond out there, but this PR does technically break BC. You might need to be careful when landing (if there are any internal users that are broken)